_data/faq_categories.yml

```diff
@@ -1,4 +1,6 @@
 # The order of the categories in this file (first-to-last) determines their
 # order in the UI (top-to-bottom)
-- title: Category 1
-- title: Category 2
+- title: General
```

Comment:
Headings should generally be sentence case: https://developers.google.com/style/headings So, for example, "Installation and deployment".